### ISSUE TITLE
tests: -m 26200 = OpenEdge Progress Encode code style

### ISSUE THREAD

**Issue by philsmd** [hashcat/hashcat#2840]
The file `tools/test_modules/m26200.pm` uses a little bit strange indentation and code style (if-else).

This is just a reminder that we should probably change this code to make it look like our other perl code (code style/formatting).

Anybody volunteer to make these small changes ? it's probably just a few lines of changes (if-else branches etc).

Thanks

**Comment by matrix** [comment]
Hi @philsmd, put a "bug" label on these things it seems exaggerated :)
The code seems to work, you should modify it directly without opening an issue right?
Please remove "bug" label here also on #2841 
Thanks

**Comment by jsteube** [comment]
I think the label was added automatically by the template selection

**Comment by philsmd** [comment]
Yeah, that's totally correct. I didn't intend to add the `bug` label... I could only select either a `feature request` or a `bug` template.

of course this "issue" was always meant to be just a nice-to-fix or nice-to-change feature because the code looked a little bit different from our other perl unit test code (but also this wouldn't be a major problem, don't get me wrong).

I just opened a couple of issues that day (see github issues: https://github.com/hashcat/hashcat/issues?q=is%3Aissue+author%3Aphilsmd) and just wanted to "report" all the minor problems I've seen lately (all of them were NOT really critical, of course).

I've now added a minor fix with the commit above. Thanks
